web/edit/db/index.php

```diff
@@ -15,6 +15,7 @@
 // Edit as someone else?
 if (($_SESSION['userContext'] === 'admin') && (!empty($_GET['user']))) {
     $user=escapeshellarg($_GET['user']);
+    $user_plain=htmlentities($_GET['user']);
```

Comment:
i cant find any use of user_plain var, probaly i missed something?

Reply:
$user_plain is used in templates for displaying the user name in the datbasename